opba-api-security/src/main/java/de/adorsys/opba/api/security/domain/SignData.java

```diff
@@ -0,0 +1,92 @@
+package de.adorsys.opba.api.security.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import lombok.extern.slf4j.Slf4j;
+
+import java.time.OffsetDateTime;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class SignData<T> {
+    private final ObjectMapper localMapper = buildObjectMapper();
+    private UUID xRequestId;
+    private OffsetDateTime requestDateTime;
+    private final T additionalFields;
+
+    public SignData(UUID xRequestId, OffsetDateTime requestDateTime) {
```

Comment:
It is better to use Instant without any timezone as this time represents a concrete moment on the time scale (in UTC TZ). Timezone complicates everything unnecessarily

Reply:
Instant.java is used instead of OffsetDateTime